src/controllers/__tests__/item.test.js

```diff
@@ -94,12 +94,40 @@ const noDescriptionItem = {
   type: 'Stationery',
 };
 
+const mockUser = {
+  username: 'chandler.bing',
+  firstName: 'Chandler',
+  lastName: 'Bing',
+  email: 'chandlerbing@gmail.com',
+  password: 'password1234',
+  passwordConfirm: 'password1234',
+  address: 'Central Perk, New York',
+  acceptTerms: true,
+};
+
 describe('Items Endpoints', () => {
   beforeAll(async () => {
     await clearDatabase();
+    const response = await request(server)
+      .post('/api/auth/signup')
+      .set('Content-Type', 'application/json')
+      .send(mockUser);
+    /* eslint-disable no-underscore-dangle */
```

Comment:
I think this rule was added to eslintrc file, so you don't need to set it individually anymore.